source/pdo_sqlsrv/pdo_dbh.cpp

```diff
@@ -1305,6 +1325,9 @@ int pdo_sqlsrv_dbh_quote( pdo_dbh_t* dbh, const char* unquoted, size_t unquoted_
 // This method is not implemented by this driver.
 pdo_sqlsrv_function_entry *pdo_sqlsrv_get_driver_methods( pdo_dbh_t *dbh, int kind TSRMLS_DC )
 {
+	if(!dbh)
+		return NULL;
+
```

Comment:
These checks for if(!dbh) is not necessary since dbh is always passed in from the pdo side and it is always not null